tokio/src/stream/mod.rs

```diff
@@ -303,7 +303,7 @@ pub trait StreamExt: Stream {
     /// As values of this stream are made available, the provided function will
     /// be run on them. If the predicate `f` resolves to
     /// [`Some(item)`](Some) then the stream will yield the value `item`, but if
-    /// it resolves to [`None`] then the next value will be produced.
+    /// it resolves to [`None`] then the next value will be skipped.
```

Comment:
```suggestion
    /// it resolves to [`None`], that value is skipped.
```

Reply:
Well, to be consistent with tense of the previous part of the sentence `stream will yield` I think it should be in the same, future, tense. 

"stream will yield ... or will skip"

Reply:
I kinda want to avoid using "next value", as I feel it is a bit ambiguous. Is it talking about the current item, or the _next_ item? I feel like it should be talking about the current item. So I suggested a wording that didn't use the word next.

Changing it to future tense sounds good :+1:

Reply:
I think `, the value will be skipped.` would be a good compromise  👯 After all, "the value" already appeared in the sentence so it is clear that we are talking about the same value again.